{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"secError"},{"kind":"text","text":"("},{"kind":"externalParam","text":"status"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"OSStatus","preciseIdentifier":"c:@T@OSStatus"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/dvesecurity\/keychainerror\/secerror(status:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/DVESecurity\/documentation\/DVESecurity\/KeychainError\/secError(status:)","interfaceLanguage":"swift"},"abstract":[{"type":"codeVoice","code":"SecKeychain"},{"type":"text","text":" operation failed with an error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"secError"},{"kind":"text","text":"("},{"kind":"externalParam","text":"status"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"OSStatus","preciseIdentifier":"c:@T@OSStatus"},{"kind":"text","text":")"}],"title":"KeychainError.secError(status:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:11DVESecurity13KeychainErrorO03secC0yACs5Int32V_tcACmF","modules":[{"name":"DVESecurity"}]},"hierarchy":{"paths":[["doc:\/\/DVESecurity\/documentation\/DVESecurity","doc:\/\/DVESecurity\/documentation\/DVESecurity\/KeychainError"]]},"references":{"doc://DVESecurity/documentation/DVESecurity/KeychainError":{"role":"symbol","title":"KeychainError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KeychainError"}],"abstract":[{"type":"text","text":"The type for errors thrown by Keychain methods."}],"identifier":"doc:\/\/DVESecurity\/documentation\/DVESecurity\/KeychainError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"KeychainError"}],"url":"\/documentation\/dvesecurity\/keychainerror"},"doc://DVESecurity/documentation/DVESecurity":{"role":"collection","title":"DVESecurity","abstract":[{"type":"text","text":"Secure your app data with cryptographic functions and by using keychains."}],"identifier":"doc:\/\/DVESecurity\/documentation\/DVESecurity","kind":"symbol","type":"topic","url":"\/documentation\/dvesecurity"},"doc://DVESecurity/documentation/DVESecurity/KeychainError/secError(status:)":{"role":"symbol","title":"KeychainError.secError(status:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"secError"},{"kind":"text","text":"("},{"kind":"externalParam","text":"status"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"OSStatus","preciseIdentifier":"c:@T@OSStatus"},{"kind":"text","text":")"}],"abstract":[{"type":"codeVoice","code":"SecKeychain"},{"type":"text","text":" operation failed with an error."}],"identifier":"doc:\/\/DVESecurity\/documentation\/DVESecurity\/KeychainError\/secError(status:)","kind":"symbol","type":"topic","url":"\/documentation\/dvesecurity\/keychainerror\/secerror(status:)"}}}